### ISSUE TITLE
`check-gmp` always fails, even though gmp is installed (Cygwin)

### ISSUE THREAD

**Issue by kernaltrap8** [ps3dev/ps3toolchain#116]
Hello,
Im trying to install the toolchain to build PSL1GHT. I don't dual boot, so im using Cygwin since it's open-source. When running `toolchain.sh` in bash, it runs for a bit and then says this: 
<img width="542" alt="image" src="https://user-images.githubusercontent.com/94473358/206207318-21b12983-5c7a-4e45-8b67-6fefc113df86.png">
I have GMP installed, I even broke Ruby to install it. 
![image](https://user-images.githubusercontent.com/94473358/206207415-c10848c6-d64a-4bf0-89c7-d65e374f07c0.png)
Any help is appreciated,
James

**Comment by zeldin** [comment]
I don't use Cygwin (I don't dual boot either :), but I suspect the `libgmp-devel` package is the one you need.
https://cygwin.com/packages/summary/libgmp-devel.html

If you end up with a list of Cygwin packages you needed to install, I'm sure we could add it to the readme to help other Cygwin users.

**Comment by JamesIsWack** [comment]
> I don't use Cygwin (I don't dual boot either :), but I suspect the `libgmp-devel` package is the one you need. https://cygwin.com/packages/summary/libgmp-devel.html
> 
> If you end up with a list of Cygwin packages you needed to install, I'm sure we could add it to the readme to help other Cygwin users.

Hello,
Seems installing `libgmp-devel` fixed the `gmp` issue, but now it gets stuck on `libelf`
![image](https://user-images.githubusercontent.com/94473358/206465334-1a258747-fba9-4f3f-a9fa-a51c52cc1acf.png)

**Comment by JamesIsWack** [comment]
> So, since you got the same problem with `libelf` that you got with `gmp`, maybe you should try the same solution (install the `-devel` package)? Just a thought...

yup, seems installing the `devel` packages of the dependencies works. add to the readme that Cygwin users must install the `devel` packages as well as the other dependencies. 

now i got a another issue. 
![image](https://user-images.githubusercontent.com/94473358/206519249-d6145be7-e5f6-46d2-9ecc-d2198c8a85fd.png)

**Comment by bucanero** [comment]
well, the error is clear, you need to set the `$PS3DEV` environment variable before running the script.

You can: `export PS3DEV=/usr/local/ps3dev`
or try running `toolchain-sudo.sh`
